### PR TITLE
[enhancement][fullstack] Improve QR experience and modifications to API routes

### PR DESCRIPTION
**Pull request by WilhelmWillie**
Changes up the UI/UX for the scanning experience in a good way -- scan page will use up the entire real estate of the window so users don't have to scroll

Also, now scans for more than just checkIn -- this can now be used for contributions

### CODE REVIEW

File: api/live.js [hunk 1]
```diff
@@ -51,15 +55,38 @@ async function handleContrib(userId, req, res) {
...
     } catch (e) {
-      return res.status(500).json({ error: e.message });
+      return res.status(500).json({ message: e.message });
     }
   }
 }
 
 async function handleCheckin(userId, req, res) {
   try {
+    const profile = await models.HackerProfile.findOne({
```
Comment (WilhelmWillie) [inline api/live.js]: moved this chunk of code up here because we want to do a status check before we attempt to create anything else

**Comment by jzeiders** [comment]
This is going to conflict with the toast PR, not sure what you want to do

**Comment by WilhelmWillie** [comment]
oh yeah i did the toast PR after this one, once this one is fixed/merged then most toast conflicts should be gone

**Comment by jzeiders** [comment]
Duplicate on #249

**Pull request closed without merging**